### ISSUE TITLE
file upload >5MB

### ISSUE THREAD

**Issue by pietz** [jnsougata/filebox#1]
Hey there,

I'm currently struggling to realize a >5MB upload using the deta python api. you've seemed to build it in Go, which I sadly dont know. Any pointers in plain english how you did it?

thanks!

**Comment by jnsougata** [comment]
actually I am uploading the file from the client side directly, not using micro as middleware. cause if i try to implement chunked upload using micro then highest size of the chunk I can send is 5.5MB but drive supports chunks size of 10MB so it's not possible ig. you can try with 5MB chunks but idk if that will work or not

**Comment by pietz** [comment]
thanks! ill try that.
